### ISSUE TITLE
Juju commands generation results in systemic misformatted '-m' option

### ISSUE THREAD

**Issue by pmatulis** [canonical-docs/canonical-doc-utils#22]
For the Juju comands generation function, there is a mess-up whereby every command's help has its `-m` option improperly formatted:

```
Options:
-m, --model (= "")
Model to operate in. Accepts [:]
```

This should instead look like:

```
Options:
-m, --model (= "")
Model to operate in. Accepts [<controller name>:]<model name>
```

**Comment by evilnick** [comment]
This is fixed in #20, though I haven't updated the snap yet

**Comment by evilnick** [comment]
Fix released
